docs/source/conf.py

```diff
@@ -321,3 +321,5 @@
     # Ignore local URLs when validating external links
     r"http://localhost:\d+/?",
 ]
+
+autodoc_typehints = "none"
```

Comment:
To suppress the `py:class` warnings. If `autodoc_typehints` is False, sphinx doesn't put type hints in the function signature.

<img width="975" alt="Screen Shot 2020-12-03 at 23 00 57" src="https://user-images.githubusercontent.com/17039389/101031503-f7a76a80-35bc-11eb-83f2-6f8f95a15516.png">